_info/channels/enchanted

```diff
@@ -1,5 +1,5 @@
 <?Flute:🎺> You have been enchanted! <?Flute:🎺> 
 
-You have been enchanted by the Flute Player(s)! You do __not__ win with the Flute team!
-Once everyone is enchanted, the Flute team win.
-If you have an investigative ability it might be affected by being enchanted.
+You have been Enchanted by the flute player(s)! You do __not__ win with the flute team!
```

Comment:
I dont know if it applies here? This is a weird message that doesn't clearly have a title and "Enchanted" mid-sentence looks very strange to me

Reply:
i agree. this was one i did "in case"